### PR TITLE
docs: Migrate from quilt to kelda

### PR DESCRIPTION
**Pull request by ejj**

### CODE REVIEW

File: docs/_HowQuiltWorks.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 
-The key idea behind Quilt is a blueprint: a blueprint describes every aspect of
+The key idea behind Kelda is a blueprint: a blueprint describes every aspect of
 running a particular application in the cloud, and is written in JavaScript.
-Quilt blueprints exist for many common applications.  Using Quilt, you can run
+Kelda blueprints exist for many common applications.  Using Kelda, you can run
 one of those applications by executing just two commands on your laptop:
 
-![Quilt Diagram](Quilt_Diagram.png)
+![Kelda Diagram](Kelda_Diagram.png)
```
Comment (kayousterhout) [inline docs/_HowQuiltWorks.md]: Can you also rename this file? (currently in docs/source/images/Quilt_Diagram.png)

File: docs/index.html.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-      Quilt Source Code
+      Kelda Source Code
     </a>
   - Documentation Powered by <a href='https://github.com/tripit/slate'>Slate</a>
 
 includes:
   - Introduction
   - GettingStarted
-  - HowQuiltWorks
+  - HowKeldaWorks
```
Comment (kayousterhout) [inline docs/index.html.md]: Rename this file (currently docs/_HowQuiltWorks.md)

File: docs/index.html.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
   - BlueprintWritersGuide
   - BlueprintAPI
   - jsdoc
-  - QuiltCLI
+  - KeldaCLI
```
Comment (kayousterhout) [inline docs/index.html.md]: Rename docs/_QuiltCLI.md to docs/_KeldaCLI.md

**Comment by kayousterhout** [comment]
The docs build currently fails because of a different issue, which I fixed in this PR: https://github.com/kelda/kelda/pull/1180

File: docs/_Dev.md [hunk 4]
```diff
@@ -202,15 +202,15 @@ Commits authored by more than one person should have a `Co-Authored-By:` tag at
...
+Patches are submitted for inclusion in Kelda using a Github Pull Request.
 
 A pull request is a collection of well formed commits that tie together
 in some theme, usually the larger goal they're trying to achieve.  Completely
 unrelated patches should be included in separate pull requests.
 
 Pull requests are reviewed by one person: either by a committer, if the code was
 submitted by a non-committer, or by a non-committer otherwise. You do not
-need to choose a reviewer yourself; [quilt-bot](https://github.com/quilt-bot)
+need to choose a reviewer yourself; [kelda-bot](https://github.com/kelda-bot)
```
Comment (kklin) [inline docs/_Dev.md]: The URL is actually https://github.com/kelda/bot

File: docs/_HowQuiltWorks.md (hunk 1 above)
Comment (ejj) [reply docs/_HowQuiltWorks.md]: 👍

File: docs/index.html.md (hunk 2 above)
Comment (ejj) [reply docs/index.html.md]: 👍

File: docs/_Dev.md (hunk 4 above)
Comment (ejj) [reply docs/_Dev.md]: 👍

**Pull request merged**